Parsers/ASimDns/Parsers/ASimDnsInfobloxNIOS.yaml

```diff
@@ -26,49 +26,47 @@ ParserQuery: |
     Syslog
     | where not(disabled)
     | where ProcessName == "named" and SyslogMessage has_all ("client", "query:", "response:")
-    | parse SyslogMessage with
-        "client "
-        SrcIpAddr:string "#" 
-        SrcPortNumber:int " "
-        NetworkProtocol:string ": query: "
-        DnsQuery:string " "
-        DnsQueryClassName:string " "
-        DnsQueryTypeName:string " "
-        "response: "
-        rest_of_string:string
-    | extend
-        // -- multi-line mode (?m) needed in regex since samples included new lines. Might not exist in real logs
-        extract = extract_all(@"^(\w+) ([\w\+]+?)(?: |$)(?m)(.*?$)?", rest_of_string)[0]
-    | extend 
-        DnsResponseCodeName = tostring(extract[0]),
-        DnsFlags = tostring(extract[1]),
-        DnsResponseName = tostring(extract[2])
-    | extend 
-        EventSubType = "response"
-    | project-away extract, rest_of_string,SyslogMessage, ProcessName, ProcessID, Facility, SeverityLevel, HostName
+    | parse SyslogMessage with *
+        "client " SrcIpAddr: string
+        "#" SrcPortNumber: string
+        " " NetworkProtocol: string
+        ": query: " DnsQuery: string
+        " " DnsQueryClassName: string
+        " " DnsQueryTypeName: string
+        " response: " DnsResponseCodeName: string
+        " " DnsFlags: string
+    | extend DNSResourceRecordIndex= indexof(DnsFlags, " ")
+    | extend DNSResourceRecord =iif(DNSResourceRecordIndex != "-1", substring(DnsFlags, DNSResourceRecordIndex), "")
```

Comment:
The field name is DnsResponseName. Best to also use it your parser.

Reply:
Changed column name from 'DNSResourceRecord' to 'DnsResponseName'.